src/Microsoft.AspNet.Mvc.TagHelpers/LinkTagHelper.cs

```diff
@@ -66,16 +70,22 @@ public class LinkTagHelper : TagHelper
         private enum Mode
         {
             /// <summary>
+            /// Just adding a file version for the src.
+            /// </summary>
+            FileVersion = 0,
+            /// <summary>
             /// Just performing file globbing search for the href, rendering a separate &lt;link&gt; for each match.
             /// </summary>
-            GlobbedHref = 0,
+            GlobbedHref = 1,
             /// <summary>
             /// Rendering a fallback block if primary stylesheet fails to load. Will also do globbing for both the
             /// primary and fallback hrefs if the appropriate properties are set.
             /// </summary>
-            Fallback = 1,
+            Fallback = 2,
         }
 
+        private FileVersionProvider FileVersionProvider { get; set; }
```

Comment:
nit: can't see how the property (and its generated methods) adds anything over a `private` field